package.json

```diff
@@ -54,6 +58,8 @@
     "react-addons-pure-render-mixin": "^15.6.2",
     "react-dom": "^15.6.2",
     "react-with-styles-interface-aphrodite": "^5.0.0",
+    "react-with-styles-interface-css": "^4.0.2",
```

Comment:
Since this is going to be used by the consumer as part of the initialize script, it will need to be part of the regular dependencies